js/site-nav.js

```diff
@@ -7,18 +7,20 @@ var $ = require('jquery');
 function SiteNav(selector) {
   this.$body = $(selector);
   this.$toggle = this.$body.find('.js-nav-toggle');
+  this.$openSublist = null;
 
   this.assignAria();
 
   this.$toggle.on('click', this.toggle.bind(this));
-  this.$body.on('change', '.js-toggle', this.handleChange.bind(this));
-  this.$body.on('mouseenter', 'li:has(.js-toggle)', this.handleMouseEnter.bind(this));
-  this.$body.on('mouseleave', 'li:has(.js-toggle)', this.handleMouseExit.bind(this));
+  this.$body.on('click', '.js-sublist-toggle', this.toggleSublist.bind(this));
+  this.$body.on('mouseenter', '.js-sublist-parent', this.toggleSublist.bind(this));
+  this.$body.on('mouseleave', '.js-sublist-parent', this.toggleSublist.bind(this));
+  $(document.body).on('focusin', this.handleFocus.bind(this));
 }
 
 SiteNav.prototype.assignAria = function() {
   var $subLists = this.$body.find('ul ul');
-  var $links = this.$body.find('.js-nav-drop-link');
+  var $links = this.$body.find('.js-sublist-toggle');
```

Comment:
This variable name is a little confusing (to me, not being familiar with the code). How about either renaming, or skipping the assignment entirely:

```javascript
this.body.find('.js-sublist-toggle').attr(...);
```